release_notes/374-release-notes.adoc

```diff
@@ -0,0 +1,321 @@
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+- NA: Not applicable
+
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+| Feature | {product-title-short} 3.72 |{product-title-short} 3.73 | {product-title-short} 3.74
+
+| Clair scanner version 2
+| GA
+| GA
+| DEP
+
+| `vulns` fields of `storage.Node` object in response payload of `v1/nodes`
+| DEP
+| DEP
+a|
+* DEP in {product-title-short} 3.74
+* REM in Advanced Cluster Security Cloud Service (ACSCS) (Field Trial)
+
+| `/v1/cves/suppress` and `/v1/cves/unsuppress`
+| DEP
+a|
+* DEP in {product-title-short} 3.73
+* REM in ACSCS
+a|
+* DEP in {product-title-short} 3.74
+* REM in ACSCS
+
+| Permission `ClusterCVE`
+| DEP
+| DEP
+| REM
+
+| Permission `Policy`
+| GA
+| GA
+| DEP
+
+| Permission `VulnerabilityReports`
+| GA
+| GA
+| DEP
+
+| Permission `Role`
+| GA
+| GA
+| DEP
+
+| `ScopeManager` default role
+| GA
+| GA
+| DEP
+
+| `BuildDate` attribute
+| GA
+| GA
+| DEP
+|====
+
+[id="deprecated-features-374"]
+=== Deprecated features
+
+The following sections describe the new permissions and indicate the deprecated permissions that will be removed in a future release.
+
+[id="removed-attribute-374"]
+==== Removed attribute
+
+The product `BuildDate` attribute is deprecated and is planned to be removed in the {product-title-short} version 4.0 release. The `/debug/versions.json` endpoint and `roxctl version --json` command will no longer return this attribute.
+
+Actions you need to take: No action is required.
+
+[id="permissions-changes-374"]
+==== Permissions changes
+
+To continue to simplify access control management, {product-title-short} will group some permissions in permission sets. As a result, the following changes are upcoming:
+
+* The permission `WorkflowAdministration` will deprecate the permissions `Policy` and `VulnerabilityReports`.
+* The permission `Access` will deprecate the permissions `Role`.
+* The default role `Scope Manager` will be removed.
+* The permission `WorkflowAdministration` will replace `Policy` and `VulnerabilityReports` in permission sets in the {product-title-short} version 4.1 release. During the migration of the permission sets within the {product-title-short} version 4.1 release, the `WorfklowAdministration` permission will have the lowest access permission granted for either `Policy` or `VulnerabilityReports`. For example, a permission set with WRITE Policy and READ `VulnerabilityReports` access will have READ `WorkflowAdministration` access after the migration that will occur with the {product-title-short} version 4.1 release. These changes in access can lead to unwanted side effects and missing access rights if you do not update your permission sets before the release.
```

Comment:
I think adding this info was the result of a change in the ACS doc process due to customer request/request from the field to be notified in advance of upcoming changes, so that's why it is included in Release Notes.